<?xml version="1.0"?>
<launch>  
<arg name="read_fast" default="false" />
<arg name="read_once" default="false" />
<arg name="repeat_delay" default="0.0" />
<param name="frame_id" value="rfan" />
<param name="DateGrade"  value="5.0_format"/>
<param name="model" value="R-Fans-16" />
<node pkg="rfans_driver" type="driver_node" name="rfans_driver" output="screen">
  <param name="advertise_name" value="rfans_packets" />
  <param name="control_name" value="rfans_control"/>
  <param name="device_ip" value="192.168.0.3" />
  <param name="device_port" value="2014" />
  <param name="rps" value="10"/>
  <param name="readfile_path"  value=""/>
  <param name="cfg_path"  value=""/>
  <param name="save_xyz"  value="false"/>
  <param name="OutExport_path"  value=""/>
  <param name="use_double_echo" value="false"/>
  <param name="use_gps" value="false"/>
  <param name="read_fast" value="$(arg read_fast)" />
  <param name="read_once" value="$(arg read_once)" />
  <param name="repeat_delay" value="$(arg repeat_delay)" />
  <param name ="cut_angle_range" value="360.0"/>
</node>

 <!--<node pkg="rfans_driver" type="u_coordinate" name="u_coordinate" output="screen"></node>-->
  <!--<node pkg="rfans_driver" type="imuPub" name="imuPub" output="screen"></node>-->
 <!--<node pkg="rviz" type="rviz" name="rviz"
 args=" -d $(find rfans_driver)/single_Rviz_cfg.rviz"/>-->
</launch>
